pyomo/solvers/tests/checks/test_GAMS.py

```diff
@@ -162,5 +162,53 @@ def test_keepfiles_gms(self):
 
             shutil.rmtree(tmpdir)
 
+
+class GAMSTests(unittest.TestCase):
```

Comment:
The additional tests should be added to the GAMSTest class above instead of creating a new class. This new class actually overwrites the tests above, which I verified in the travis log file (https://travis-ci.org/Pyomo/pyomo/jobs/329755594). Alternatively the class name can be changed for the new class.

Reply:
Yeah noticed that too, sorry.  Got the tests running locally now so next time will be smoother.